package.json

```diff
@@ -1,5 +1,7 @@
 {
   "name": "ftdomdelegate",
```

Comment:
This seems like a bug in obt to me. Maybe we only check for package.json conformance if the origami type is component and not for library?